extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss

```diff
@@ -31,28 +32,40 @@
 }
 
 #CreateNewWiki {
-	background: white;
+	background: $wds-color-white;
 	box-sizing: border-box;
 	border: 1px solid $wds-color-faint-gray;
-	color: #747474;
-	font-size: 12px;
-	font-weight: normal;
+	color: $wds-color-dark-gray;
+	font-size: $wds-typescale-size-base;
+	line-height: $wds-line-height;
 	margin: 4px auto 0;
 	padding: $content-spacing-regular 0 $cnw-bottom-bar-size;
 	position: relative;
 	width: $cnw-size;
 
+	::-webkit-input-placeholder, :-moz-placeholder, ::-moz-placeholder, :-ms-input-placeholder {
```

Comment:
You can use placeholder mixin (https://github.com/Wikia/app/blob/dev/skins/shared/mixins/placeholder.scss)